addons/website_sale_picking/static/src/js/checkout_form.js

```diff
@@ -0,0 +1,49 @@
+/** @odoo-module */
+
+import publicWidget from 'web.public.widget';
+import 'website_sale_delivery.checkout';
+
+publicWidget.registry.websiteSaleDelivery.include({
+    start: function () {
+        this.paymentOptions = document.querySelectorAll('.o_payment_option_card input[type=radio]');
+        return this._super.apply(this, ...arguments);
+    },
+
+    /**
+     * Hides or shows a payment option card.
+     * @param node the input element of the payment option card
+     * @param enabled whether to show or hide the card
+     * @private
+     */
+    _setEnablePaymentOption(node, enabled=false){
+        if(enabled){
+            node.parentNode.parentNode.classList.remove('d-none');
+            try{
+                $(node).click();
+            }catch (_){ // During a tour, if the button is clicked very quickly, it seems the code handling the payment click is not yet ready ?
+                node.checked = true;
+            }
```

Comment:
We try and avoid code test-specific. Isn't there a way to add an extra-trigger in the tour that ensure the payment option is enabled ?